02-spatial-data.Rmd

```diff
@@ -177,27 +177,30 @@ text(x = ncentre_num[1], y = ncentre_num[2], labels = "Nigeria")
 ### Further work
 
 **sf** makes R data objects more closely aligned to the data model used in GDAL and GEOS, in theory making spatial data operations faster.
-The work here provides a taste of the way how **sf** operates but there is much more to learn (see Chapter \@ref(spatial-data-operations)).
-There is a wealth of information in the highly recommended vignettes of the package.
+The work here provides a taste of the way **sf** operates but there is much more to learn (see Chapter \@ref(spatial-data-operations)). And there is also a wealth of information in the highly recommended vignettes of the package.
 
-As a final exercise, we'll see how to do a spatial overlay in **sf** by first converting the countries of the world into centroids and then subsetting those in Asia.
-<!-- Maybe rephrase. I was instantly thinking of an actual spatial overlay (`over`) but you mereley refer to a map overlay. -->
+As a final exercise, we'll see one way of how to do a spatial overlay in **sf**. First, we convert the countries of the world into centroids, and then subset those in Asia. Finally, the `summary`-command tells us how many centroids (countries) are part of Asia (43) and how many are not (134).
+
+```{r}
+world_centroids = st_centroid(world)
+sel_asia = st_intersects(world_centroids, asia, sparse = FALSE)
+summary(sel_asia)
+```
+
+Note: another way of achieving the same result is with a GEOS function for identifying spatial overlay, which we'll cover in more datail in Chapter \@ref(spatial-data-operations).
```

Comment:
Sorry, my mistake, you are actually performing a spatial overlay but I got confused because you said you were about to do a spatial overlay but then you plot the continents and add the country coordinates. That's why I rearranged the order. I think that makes it clearer.